README.md

```diff
@@ -27,7 +27,9 @@ Add the `InlineActionsModelAdminMixin` to your `ModelAdmin`.
 If you want to have actions on your inlines, add the `InlineActionMixin` to your `InlineModelAdmin`.
```

Comment:
If you can please fix `InlineActionMixin`, it's `InlineActionsMixin` with a `s`.